workflow/workflow-demos/processes/BusinessData/VerificationProcess.p.json

```diff
@@ -100,7 +97,16 @@
           ],
           "map" : {
             "out.id" : "param.id"
-          }
+          },
+          "code" : [
+            "import ch.ivyteam.ivy.bpm.error.BpmError;",
+            "",
+            "if (param.id.length() == 0) {",
+            "  BpmError.create(\"verification:error\")",
+            "    .withMessage(\"This process should be called with a dossier id.\")",
+            "    .throwError();",
+            "}"
+          ]

```

Comment:
This process is not listed in the starts list, but it can be started via the business cases dossier showcase. 
However, it need to have a dossier id. Without it it makes no much sense and stuff like the task title will not be correctly evaluated.
So just hard prevent this task to run if the id is not given via parameter.